src/components/AgmExtension/AgmExtensionEvaluation.vue

```diff
@@ -88,5 +106,13 @@ import { AgmExtEvalIF } from '@/interfaces'
 })
 export default class AgmExtensionEvaluation extends Vue {
   @Prop({ required: true }) readonly data!: AgmExtEvalIF
+
+  @Prop({ default: '' }) readonly extensionDurationText!: string
+
+  @Prop({ default: '' }) readonly dueDateText!: string
```

Comment:
I was rather hoping that THIS component could figure these strings instead of having them passed in.

It's the idea of separation of concerns: don't involve extra components in computations if they don't need to know. In this case, the parent component is simply a framework for the data, not the source of these particular props. Karim's component does all the computations. And THIS component can react to Karim's properties and display the proper strings.

Another way to look at it is trying to make self-contained components with as-simple-as-possible interfaces to the rest of their world (props/events).

Reply:
Sev, here's what I was thinking:

This component's non-eligible text will be set in the parent after my logic is done. This is since I believe the text depends on which scenario didn't pass in my component. I might need to follow-up with Yui or Mihai about that though. Not sure if it's the same text for ineligibility for all scenarios or not.

I guess what you're thinking is instead of letting the parent decide the text, we remove these props and have this component set the text depending on whether eligible is true or false? That would work only if the text is the same across all scenarios like I mentioned above though if that makes sense.

Reply:
Kinda the second one.

For audit purposes, I'm expecting your component to set a value(s) indicating which scenario the business is in (which would get saved in the filing JSON). Then, THIS component can simply look at that value(s) and show the proper text.

Reply:
I see. OK.

@PaulGarewal Can you please follow-up with Yui/Mihai and ask them what would the text be depending on the ineligibility scenario? The UXPin only shows one.

Reply:
Fixed, thanks.